### ISSUE TITLE
Overwrite with CLI args

### ISSUE THREAD

**Issue by markus-hinsche** [kbressem/trainlib#25]
I keep having the same changes for my local vs. my workstation setup (only the latter has all the data and a GPU)

![image](https://user-images.githubusercontent.com/1809832/193103231-2c7d2297-741a-4886-bff0-1137482512f6.png)

Ideally I could do this locally: 
`python raidiance/train_pathology.py --config raidiance/configs/config_pathology.yaml --data.dataset_type non_persistent`

This way I wouldn't have to keep local changes around which I have to be careful not to commit

**Comment by kbressem** [comment]
Yes, I have also thought about this recently. In simple training routines, I added the `--debug` flag to overwrite the config, but wished it could be extended even more. `monai.bundle` allows exactly for this functionality. I believe it is necessary to write a better config parser for this. We could also solve #5 with this. Furthermore, I wanted to refine possibilities for hyperparameter tuning, so instead of a single value you can pass a range in the config. 

I think the best thing is to start a backlog for `trainlib` and start including these things systematically.
